### PR TITLE
Add routing doc

### PR DESCRIPTION
**Pull request by gund**
Closes [HRZ-2149](https://spryker.atlassian.net/browse/HRZ-2149)

[HRZ-2149]: https://spryker.atlassian.net/browse/HRZ-2149?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: _drafts/oryx/routing.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+}
+```
+
+Then you can provide routes for Lit applications using DI:
+
+```ts
+import { appBuilder } from '@spryker-oryx/core';
+import { provideLitRoutes } from '@spryker-oryx/router/lit';
+
+const routes = [{ path: '/', render: () => html`<my-page></my-page>` }];
```
Comment (dunqan) [inline _drafts/oryx/routing.md:49]: maybe we can second route example with parameter here?

**Review by dunqan — DISMISSED** [review]

File: _drafts/oryx/routing.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Routing
+
+Routing is an essential feature for any web application, allowing users to navigate between different pages or components within the application. The `@spryker-oryx/router` package
+provides routing infrastructure for your application, including a `RouterService` for interacting with the router and framework-specific integrations for providing actual routing config.
```
Comment (andriitserkovnyi) [inline _drafts/oryx/routing.md:4]: Please split the last sentence into two. The last part, in particular, doesn't make sense for me

File: _drafts/oryx/routing.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+This guide will walk you through the process of setting up routing in your application, including how to add the `RouterFeature` to your application, render the router outlet, and provide routes using Dependency Injection (DI).
+
+`@spryker-oryx/router` provides infrastructure for routing capabilities. It gives you mainly the `RouterService` which is a generic way for navigating in your application.
```
Comment (andriitserkovnyi) [inline _drafts/oryx/routing.md:7]: It's the same info as in the first paragraph. Where would it make more sense?

File: _drafts/oryx/routing.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  { path: '/', render: () => html`<my-page></my-page>` },
+  { path: '/example/:id', render: ({ id }) => html`<my-example-page id=${id}></my-example-page>` },
+];
+
+appBuilder().withProviders(provideLitRoutes({ routes }));
+```
+
+Please note that this is a basic example and you may need to adjust the routing config
+to fit your specific use case.
+Lit routes using standard [lit router library](https://github.com/lit/lit/tree/main/packages/labs/router#readme) so you can read it's documentation for more information on how to configure routes.
```
[the remark this answers is not in the document]
Comment (gund) [reply _drafts/oryx/routing.md:59]: There is already a link to docs https://github.com/lit/lit/tree/main/packages/labs/router#readme

File: _drafts/oryx/routing.md (hunk 3 above)
Comment (gund) [reply _drafts/oryx/routing.md:7]: I guess we can remove the second sentence from the last paragraph to reduce duplicated info.

Comment (gund) [reply _drafts/oryx/routing.md:7]: I've rearranged it and removed it from the first paragraph.

**Review by jmartinspryker — APPROVED** [review]

File: _drafts/oryx/routing.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+```
+{% info_block infoBox %}
+
+This is a basic example, and you may need to adjust the routing config to fit your specific use case.
+
+{% endinfo_block %}
+
+
+
+Lit routes using the standard [lit router library](https://github.com/lit/lit/tree/main/packages/labs/router#readme), so see its documentation for more information on how to configure routes.
```
Comment (VadymSachenko) [inline _drafts/oryx/routing.md:65]: I would suggest adding a link to the official documentation.
```suggestion
Lit routes use the standard [lit router library](https://github.com/lit/lit/tree/main/packages/labs/router#readme). For more information about configuring routes, see its documentation.
```

File: _drafts/oryx/routing.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+class RootComponent extends LitElement {
+  router = new LitRouter(this, []); // <-- You can provide some static routes here
+
+  protected override render() {
+    return this.router.outlet(); // <-- Render router outlet here
+  }
+}
+```
+
+Then, you can provide routes for Lit applications using DI:
```
Comment (VadymSachenko) [inline _drafts/oryx/routing.md:44]: Is this step optional?

File: _drafts/oryx/routing.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
+---
+title: Routing
+description:
+template: concept-topic-template
+last_updated: Apr 4, 2023
```
Comment (VadymSachenko) [inline _drafts/oryx/routing.md:5]:
```suggestion
last_updated: May 25, 2023
```

**Review by VadymSachenko — CHANGES_REQUESTED** [review]

File: _drafts/oryx/routing.md (hunk 6 above)
Comment (andriitserkovnyi) [reply _drafts/oryx/routing.md:44]: This is more of a concept section than a guide

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**